_includes/master/kubernetes/k8s_binary_init.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I think given you are thinking that people might want to use this separately, then we should deffo have help text and some comment at the top of the file indicating it's purpose (e.g. what it's actually downloading)